packages/react-core/src/components/Form/FormFieldGroupToggle.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Form/form';
+import { css } from '@patternfly/react-styles';
+import AngleRightIcon from '@patternfly/react-icons/dist/js/icons/angle-right-icon';
+import AngleDownIcon from '@patternfly/react-icons/dist/js/icons/angle-down-icon';
+import { Button } from '../Button';
+
+export interface FormFieldGroupToggleProps extends React.HTMLProps<HTMLDivElement> {
+  /** Additional classes added to the section */
+  className?: string;
+  /** Callback for onClick */
+  onToggle: () => void;
+  /** Flag indicating if the toggle is expanded */
+  isExpanded: boolean;
+  /** Aria label of the toggle button */
+  'aria-label'?: string;
+  /** Sets the aria-labelledby attribute on the toggle button element */
+  'aria-labelledby'?: string;
+  /** The id applied to the toggle button */
+  toggleId?: string;
+}
+
+export const FormFieldGroupToggle: React.FunctionComponent<FormFieldGroupToggleProps> = ({
+  className,
+  onToggle,
+  isExpanded,
+  'aria-label': ariaLabel,
+  'aria-labelledby': ariaLabelledby,
+  toggleId,
+  ...props
+}: FormFieldGroupToggleProps) => (
+  <div className={css(styles.formFieldGroupToggle, className)} {...props}>
+    <div className={css(styles.formFieldGroupToggleButton)}>
+      <Button
+        variant="plain"
+        aria-label={ariaLabel}
+        onClick={onToggle}
+        aria-expanded={isExpanded}
+        aria-labelledby={ariaLabelledby}
+        id={toggleId}
+      >
+        {isExpanded ? <AngleDownIcon aria-hidden="true" /> : <AngleRightIcon aria-hidden="false" />}
```

Comment:
If you wrap this in `span.pf-c-form__field-group-toggle-icon`, then you only need to include the right angle, and the CSS will rotate it with the expanded class on the field group parent element. That will also enable the transition animation of the icon rotating when toggling the expanded state.

Also why is `aria-hidden="false"` with one icon and not the other?